src/suggestion.rs

```diff
@@ -50,14 +69,206 @@ impl fmt::Display for Detector {
     }
 }
 
+// For long lines, literal will be trimmed to display in one terminal line.
+// Misspelled words that are too long shall also be ellipsized.
+pub fn condition_display_content(
+    terminal_size: usize,
+    _indent: usize,
+    stripped_line: &str,
+    mistake_range: Range,
+    padding_till_excerpt_start: usize,
+    marker_size: usize,
+) -> (String, usize, usize) {
+    if stripped_line.chars().count() + padding_till_excerpt_start <= terminal_size {
+        return (stripped_line.to_owned(), mistake_range.start, marker_size);
+    }
+
+    // The paddings give some space for the ` {} ...` and extra indentation and formatting:
+    //
+    //|---------------------------------------------------------------------------------------| terminal_size
+    //|-------| padding_till_excerpt_start = indent (3+line_number_digit_count) + 2 white spaces = 7usize, for this case.
```

Comment:
this needs to be revisited after as I believe there display of the sentence starts after 3 empty white spaces in the display

Reply:
Not exactly, IIRC it depends on the number of digits in the line number

Reply:
yeah, I guess this already accounts to it `line_number_digit_count`. I was referring to this `2` white spaces that I account after `| space space ..sentence..`

Reply:
I'd like to push this to a separate PR or if you could add explicit change you would like to see here :)

Reply:
I guess it makes sense if you remove and we add in separate PR. It will be more clear also what to add.